proposals/1/standardized-mod-format.md

```diff
@@ -0,0 +1,126 @@
+# Standardized Mod Format (DRAFT!)
+
+## Description Of Issue
+
+Lack of a clear specification between modloaders on a common supported format.
+
+## Solution
+
+A specification which exactly defines specific terms, and does not define legacy formats from the previous era.
```

Comment:
Why should legacy formats not be described?

Reply:
Because it'd either implicitly require support for them or be kind of a waste of space...

Reply:
Defining legacy formats as optional would allow modloaders to support old mods

Reply:
This specification does not state that modloaders can't have their own support mechanisms if they so choose.

Reply:
Resolved with b5d0e83 clarifying that these are allowed without explicitly defining their format.